### PR TITLE
DDD 47-48 Added Order History in profile page

### PR DESCRIPTION
**Pull request by Zygis0321**

### CODE REVIEW

File: ui/src/components/SummaryCards/ParcelSizeSummaryCard.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import {Grid} from "@material-ui/core";
+import Box from "../../images/box.png";
+import Letter from "../../images/letter.png";
+import ImageBox from "../ImageBox";
+
+export default function ParcelSizeSummaryCard(props) {
+    return (
+        <div className="form-wrapper summary-subform-wrapper">
```
Comment (Accir) [inline ui/src/components/SummaryCards/ParcelSizeSummaryCard.jsx]: forgot to add the class `"package-type-summary-wrapper"`

File: ui/src/components/OrderHistory.jsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    }
+
+    return(
+        <div className="form-wrapper">
+            <div className="form-inner">
+            <div className="form-header">Order History</div>
+                <List>
+                    {
+                        orders.map((order, index) => 
+                            <>
```
Comment (Accir) [inline ui/src/components/OrderHistory.jsx]: add `key` to this empty tag

File: ui/src/components/OrderHistory.jsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                                        <Grid item xs={4}>
+                                            <OrderInfo
+                                                title = "Package"
+                                                description = {getPackageSize(order).name}
+                                            />
+                                        </Grid>
+                                    </Grid>
+                                    <ListItemSecondaryAction>
+                                        <IconButton edge="end" size="small" onClick = {() => selectOrder(order)}>
+                                            <ArrowForwardIcon />
```
Comment (Accir) [inline ui/src/components/OrderHistory.jsx]: Do we really need an arrow if the whole order is already clickable?

File: ui/src/components/OrderHistory.jsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+                                        <Grid item xs={4}>
+                                            <OrderInfo
+                                                title = "Status"
+                                                description = {order.status}
+                                            />
+                                        </Grid>
+                                        <Grid item xs={4}>
+                                            <OrderInfo
+                                                title = "Recipient"
+                                                description = {order.recipient.name}
```
Comment (Accir) [inline ui/src/components/OrderHistory.jsx]: Maybe let's also display recipient's surname here

File: ui/src/components/OrderHistory.jsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+                                <ListItem 
+                                    button 
+                                    key={index}
+                                    onClick = {() => selectOrder(order)}
+                                >
+                                    <Grid container>
+                                        <Grid item xs={4}>
+                                            <OrderInfo
+                                                title = "Status"
+                                                description = {order.status}
```
Comment (Accir) [inline ui/src/components/OrderHistory.jsx]: I think we need some color on different status to make it more visible

File: ui/src/components/OrderModal.jsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+import SenderSummaryCard from "./SummaryCards/SenderSummaryCard";
+import {Dialog} from "@material-ui/core"
+import "../styles/OrderModal.css"
+
+export default function OrderModal(props){
+    return(
+        <Dialog
+            open = {props.open}
+            onClose = {() => props.onClose(props.order)}
+        >
```
Comment (Accir) [inline ui/src/components/OrderModal.jsx]: Maybe set `border-radius: 0 ` to comply with the app's style

File: ui/src/components/SummaryCards/ParcelSizeSummaryCard.jsx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+                            <li>Max length: {props.selectedPackage.length}cm</li>
+                            <li>Max width: {props.selectedPackage.width}cm</li>
+                            {props.selectedPackageType === "Box" ? <li>Max height: {props.selectedPackage.height}cm</li> : ""}
+                        </ul>
+                    </Grid>
+                </Grid>
+            </div>
+        </div>
+    )
+}
```
Comment (giplonis) [inline ui/src/components/SummaryCards/ParcelSizeSummaryCard.jsx]: I'm nitpicking a bit here, but for the future - always make sure to leave 1 blank line at the end of file. It's a convention and a good practice.

File: ui/src/components/OrderModal.jsx [hunk 8]
```diff
@@ -0,0 +1,32 @@
+import ParcelSizeSummaryCard from "./SummaryCards/ParcelSizeSummaryCard";
+import RecipientSummaryCard from "./SummaryCards/RecipientSummaryCard";
+import SenderSummaryCard from "./SummaryCards/SenderSummaryCard";
+import {Dialog} from "@material-ui/core"
+import "../styles/OrderModal.css"
+
+export default function OrderModal(props){
```
Comment (giplonis) [inline ui/src/components/OrderModal.jsx]: Just a minor thing - missing space after `(props)` 😄

Comment (Accir) [reply ui/src/components/OrderModal.jsx]: @Zygis0321 do you have prettier installed?

File: ui/src/components/OrderHistory.jsx (hunk 5 above)
Comment (Zygis0321) [reply ui/src/components/OrderHistory.jsx]: Added icons to make status more visible

File: ui/src/components/OrderModal.jsx (hunk 6 above)
Comment (Zygis0321) [reply ui/src/components/OrderModal.jsx]: Did global theme with border-radius 0

**Review by Accir — APPROVED** [review]

**Pull request merged**